modules/framework/galasa-parent/dev.galasa.framework.k8s.controller/src/main/java/dev/galasa/framework/k8s/controller/TestPodScheduler.java

```diff
@@ -533,51 +508,4 @@ private V1EnvVar createValueEnv(String name, String value) {
         return env;
     }
 
-    private V1EnvVar createConfigMapEnv(String name, String configMap, String key) {
-        V1EnvVar env = new V1EnvVar();
-        env.setName(name);
-
-        V1EnvVarSource source = new V1EnvVarSource();
-        env.setValueFrom(source);
-
-        V1ConfigMapKeySelector configMapKeyRef = new V1ConfigMapKeySelector();
-        source.setConfigMapKeyRef(configMapKeyRef);
-
-        configMapKeyRef.setName(configMap);
-        configMapKeyRef.setKey(key);
-
-        return env;
-    }
-
-    private V1EnvVar createSecretEnv(String name, String secret, String key) {
```

Comment:
This method was not being used. Removing for clarity.

Reply:
OK